astropy/nddata/nddata.py

```diff
@@ -192,12 +219,51 @@ def mask(self, value):
 
     @property
     def unit(self):
+        """
+        `~astropy.units.Unit`: Unit for the data, if any.
+        """
         return self._unit
 
     @property
     def wcs(self):
+        """
+        any type: wcs for the data, if any.
+
+        Even though nothing is enforced, using `~astropy.wcs.WCS` as ``wcs``
+        attribute is encouraged.
+        """
         return self._wcs
 
     @property
     def meta(self):
+        """
+        `dict`-like: Meta information, if any.
+        """
         return self._meta
+
+    @property
+    def uncertainty(self):
+        """
+        any type: Uncertainty in the data, if any.
+
+        Uncertainty *should* have an attribute ``uncertainty_type`` that is
+        a string. `~astropy.nddata.NDUncertainty`-subclasses are recommended,
+        if `~astropy.nddata.NDArithmeticMixin` with uncertainty propagation
```

Comment:
I wonder if this makes more sense somewhere in the docs for `NDArithmeticMixin`?